R/vdiagram-data.R

```diff
@@ -63,21 +79,15 @@ historyMeasurementsLabel <- function(reportObject) {
   }
 }
 
+#' Set default historic flags
+#' @description Sets the historic flag to false if " " is passed in.
+#' @param histFlag The historic flag field from a V Diagram.
+#' @return FALSE if " " is passed in or if the field is empty, otherwise it returns what was passed in. 
 defaultHistFlags <- function(histFlag){
-  x <- NULL
-  if (length(histFlag)==1 && histFlag == " "){
-    histFlag <- rep(TRUE, length(x))
-  }
-
-  if (isEmptyOrBlank(histFlag)){
+  
+  if (isEmptyOrBlank(histFlag) || (length(histFlag)==1 && histFlag == " ")){
     histFlag <- FALSE
   }
```

Comment:
why does a length of 1 warrant FALSE?

seems weird for the `histFlag` to be anything but a logical. Why would it have a value?